source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -75,7 +77,13 @@ The ``FindOne()`` method expects you to pass a ``Context`` type and a
 query filter. The method returns *the first document* that matches the
 filter as a ``SingleResult`` type.
 
-To learn how to access data in a ``SingleResult`` see :ref:`golang-bson-unmarshalling`.
+For a ``FindOne()`` method code example, see the :ref:`golang-find-one-example`
+section of this page. For a ``FindOne()`` code example that queries by using
+a specific ``ObjectId`` value, see the :ref:`golang-find-one-by-id` section
+of this page.
+
+To learn how to access data in a ``SingleResult``, see
```

Comment:
```suggestion
To learn how to access data from a ``SingleResult`` type, see
```

Reply:
In general, where a type/method is mentioned, explain what it is. Aka 
the ``UpdateOne()`` method
a value of type ``Result``
an instance of ``MyStruct``